index.html

```diff
@@ -17,13 +18,17 @@
             <div class="logo">
                 <h3>Dhuruv</h3>
             </div>
-            <div class="menu">
-            </div>
-            <ul class="desktop-menu">
-                <li><a href="#main-sec">Portfolio</a></li>
-                <li><a href="#about-me">About</a></li>
-                <lI><a href="#contact-form">Contact</a></lI>
+            <!-- <div class="menu">
+            </div> -->
+            <ul class="menu nav-icon desktop-menu">
+                <li><a class="menuItem nav-menu" href="#main-sec">Portfolio</a></li>
+                <li><a class="menuItem nav-menu" href="#about-me">About</a></li>
+                <lI><a class="menuItem nav-menu" href="#contact-form">Contact</a></lI>
```

Comment:
- [ ] You have done well implementing the functionality to toggle the mobile menu 📱 . I noticed a vertical scroll bar on the nav lit of the desktop site. Please fix it 👍 

![image](https://user-images.githubusercontent.com/63473584/179640296-2e0c67f9-4c64-4202-a4d3-0a5013cfce6b.png)

Reply:
done